src/gocept/sftpcopy/sftpcopy.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2006-2014 gocept gmbh & co. kg
 # See also LICENSE.txt
 
-import ConfigParser
+import configparser
```

Comment:
How does that work on Python 2 and Python 3? Do I miss something here?

I see, that in the `tox.ini` the package `configparser < 5` is used in case of Python 2. This should also be specified in the `setup.py`.

Reply:
Nice catch.